src/test/test-sizeof.c

```diff
@@ -1,16 +1,17 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 
-#include <sched.h>
 #include <stdio.h>
 #include <string.h>
 #include <sys/resource.h>
 #include <sys/socket.h>
+#include <sys/timex.h>
 #include <sys/types.h>
 
 #define __STDC_WANT_IEC_60559_TYPES_EXT__
 #include <float.h>
 
 #include "time-util.h"
+#include "cpu-set-util.h"
```

Comment:
Why did you change `sched.h` -> `cpu-set-util.h`??

Reply:
oversight. Removed in v2